curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-083.md

```diff
@@ -8,9 +8,9 @@ isHidden: true
 ## Description
 <section id='description'>
 
-The menu looks good, but other than the coffee beans background image, it is mainly just text.
+You change properties of a link when the mouse hovers them by using a <dfn>pseudo-selector</dfn> that looks like `a:hover { propertyName: propertyValue }`
```

Comment:
You change properties of a link when the mouse hovers them by using a <dfn>pseudo-selector</dfn> that looks like `a:hover { propertyName: propertyValue }`.